src/App.tsx

```diff
@@ -1,109 +1,78 @@
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { getItemsOnPage } from './utils/pagination/getItemsOnPage';
+import { getItemsOnPageCount } from './utils/pagination/getItemsOnPageCount';
+import { Pagination } from './components/Pagination';
+import { EPaginationSize } from './enums/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState<EPaginationSize>(
+    EPaginationSize.Medium,
+  );
```

Comment:
The `perPage` state is initialized with `EPaginationSize.Medium`, which suggests that `EPaginationSize` is an enum. However, the `perPage` state should be a number, as it represents the number of items per page. Ensure that the `EPaginationSize` enum values are numbers or change the state type to `number`.